spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/authService/SCAuthService.java

```diff
@@ -71,19 +76,35 @@ public void authenticate(String username, String password) {
         }
     }
 
+    /**
+     * Observable that will send current status and will send updates as subscribed
+     * @return BehaviorSubject<Integer>
+     */
     public BehaviorSubject<Integer> getLoginStatus() {
         return loginStatus;
     }
 
+    /**
+     * JSONWebToken from auth server
+     * @return String
+     */
     public String getAccessToken() {
         return authMethod.xAccessToken();
     }
 
+    /**
+     * this will void the x-access-token
```

Comment:
We're not using `x-access-token` anymore.  We're sending a token in the Authorization header as a Token type.  So maybe say it will void the JWT?